vscode-dotnet-runtime-library/src/Acquisition/IInstallationValidator.ts

```diff
@@ -3,6 +3,7 @@
 *  The .NET Foundation licenses this file to you under the MIT license.
 *--------------------------------------------------------------------------------------------*/
 
+import { IDotnetAcquireContext, IVSCodeExtensionContext } from '..';

```

Comment:
Is this needed?

Reply:
This is no longer needed, thank you, I will remove this in a upcoming change